remix-lib/test/txFormat.js

```diff
@@ -112,8 +112,10 @@ tape('ContractNestedArrayParameters - (TxFormat.buildData) - format nested array
 })
 
 function testWithNestedArrayInput (st, params, expected) {
-  txFormat.buildData('nestedArrayContractTest', context.contract, context.output.contracts, true, context.contract.abi[1], params, (error, data) => {
-    if (error) { return st.fails(error) }
+  txFormat.buildData('nestedArrayContractTest', context.contract, context.output.contracts, true, context.contract.abi[4], params, (error, data) => {
+    if (error) {
+      return st.fails(error)
+    }
     console.log(data)
```

Comment:
Why do we have this console.log ? Can we get rid of it ?

Reply:
There are multiple consoles in this file. am not very sure about the reason.